Gemfile

```diff
@@ -1,4 +1,8 @@
 source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
+gem 'mutant'
+gem 'mutant-rspec'
+gem 'mutant-license', source: 'https://oss:IG9NKHkepT1VE6CyvJiYH14GUsCHPtog@gem.mutant.dev'
```

Comment:
should it be visible to internet?

Reply:
It doesn't matter, the license is verifying git repository remote URL. So copying this value to another project will not work. It only works for this specific URL of this repository